buildtest/cli/config.py

```diff
@@ -153,6 +155,11 @@ def view_configuration(configuration, theme=None, pager=None):
     console.rule(configuration.file)
     console.print(syntax)
 
+def view_profile(configuration, theme=None, pager=None):
+    """Display the list of profile for buildtest configuration file.This implements command ``buildtest config profiles list`
+    """
+
+    console.print(configuration.file)
```

Comment:
this is not right for the implementation. You need to index the dictionary and list all the profiles. the `configuration.file` is the full path to configuration file.